views/blog.erb

```diff
@@ -0,0 +1,37 @@
+<html>
+  <head>
+    <title>Emily's Blog</title>
+    <meta charset = "utf-8">
```

Comment:
With Ruby we are used to adding a space between the `=` symbol but in HTML we want to exclude the spaces on either side of the `=`